README.md

```diff
@@ -935,10 +933,19 @@ To check the code coverage, run:
 COVERAGE=yes bundle exec rake parallel_spec
```

Comment:
Shouldn't `rake check` be the recommended approach since it also runs all the linting? It's provided by `puppetlabs_spec_helper`.

Reply:
I'm going to leave this one alone for now as out of scope for this current change set, but it'll need a proper tackling when we come back through.